spawn_operator/spawn_operator/lib/spawn_operator/k8s/proxy/deployment.ex

```diff
@@ -101,10 +101,12 @@ defmodule SpawnOperator.K8s.Proxy.Deployment do
               "actor-system" => system
             }
           },
-          "spec" => %{
-            "containers" => get_containers(embedded, system, name, host_params, annotations),
-            "terminationGracePeriodSeconds" => @default_termination_period_seconds
-          }
+          "spec" =>
+            %{
+              "containers" => get_containers(embedded, system, name, host_params, annotations),
+              "terminationGracePeriodSeconds" => @default_termination_period_seconds
+            }
+            |> maybe_put_volumes(params)
```

Comment:
I think it would be worth validating if the name attribute has been set. Just a double check

Reply:
> I think it would be worth validating if the name attribute has been set. Just a double check

And which is also equal to the one mapped in the volumeMount